x-pack/plugins/apm/public/services/rest/observability_dashboard.ts

```diff
@@ -48,7 +48,12 @@ export const fetchLandingPageData = async (
           'xpack.apm.observabilityDashboard.stats.transactions',
           { defaultMessage: 'Transactions' }
         ),
-        value: sum(transactionCoordinates.map((coordinates) => coordinates.y)),
+        value:
+          mean(
+            transactionCoordinates
+              .map(({ y }) => y)
+              .filter((y) => y && isFinite(y))
```

Comment:
I think you can do:
```suggestion
              .filter(isFinite)
```

Reply:
`isFinite` requires a number, and `y` can be `number | undefined`. That's why I didn't do what you've suggested.